src/lib/keys.js

```diff
@@ -141,27 +142,33 @@ const deriveNetworkSeedFromMnemonic = async (
   return Just(await kg.deriveNetworkSeed(mnemonic, passphrase, revision - 1));
 };
 
+/**
+ * @param {string} authToken
+ * @param {number} number
+ * @return {Maybe<string>}
+ */
+export const deriveNetworkSeedFromAuthToken = (authToken, revision, point) => {
+  const networkSeed = shas(authToken, `revision-${point}-${revision}`)
+    .toString('hex')
+    .slice(0, 32);
+  return Just(networkSeed);
+};
+
 /**
  * @return {Promise<Maybe<string>>}
  */
 export const attemptNetworkSeedDerivation = async ({
   urbitWallet,
-  wallet,
-  authMnemonic,
-  details,
+  point,
+  authToken,
   revision,
 }) => {
   if (Just.hasInstance(urbitWallet)) {
     return await deriveNetworkSeedFromUrbitWallet(urbitWallet.value, revision);
   }
 
-  if (Just.hasInstance(wallet) && Just.hasInstance(authMnemonic)) {
-    return await deriveNetworkSeedFromManagementMnemonic(
-      wallet.value,
-      authMnemonic.value,
-      details,
-      revision
-    );
+  if (Just.hasInstance(authToken)) {
+    return deriveNetworkSeedFromAuthToken(authToken.value, point, revision);
```

Comment:
The arguments for the function say `revision, point` though, not the other way around.

Should probably make ordering of these consistent with `attemptNetworkSeedDerivation` anyway.